draft-ietf-tls-tls13.md

```diff
@@ -1844,8 +1863,21 @@ A number of TLS messages contain tag-length-value encoded extensions structures.
        } Extension;
 
        enum {
+           server_name(0),
```

Comment:
The motivation here was to match the ones listed in the table that permits them in the various fields.